packages/server/public/css/index/sync_debug.css

```diff
@@ -0,0 +1,37 @@
+
+
```

Comment:
This file should be refactored to comply with Joplin's style guide. (Joplin uses [RCSS](https://ricostacruz.com/rscss/nested-components.html)).

(The RCSS guide assumes SCSS and this file is CSS, but it should still be possible to apply naming conventions, etc.)